motd/stable.md

```diff
@@ -1,3 +1,8 @@
+**June 20th, 2019**
+Power outage this Sunday at NCSA. 
+Systems at NCSA will be powered down on Sunday June 23rd due to building work. We have a number of volunteers that have offered to try and bring the systems back on Sunday night, to minimise business hour disruption. However bear in mind that they might end up needing a fix from somebody who is not available, so I can't rule out the possibility that there will be after-effects on Monday morning
```

Comment:
@gpdf has asked that we also include a link back to the full listing since the news feed only shows a few lines.  The standard wording is:
`Check [here](https://github.com/lsst-dm/lsp-landing-page/blob/master/motd/stable.md) for more details and the complete list of messages.`